presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -101,8 +103,8 @@ public void testJoin()
                                 INNER,
                                 ImmutableList.of(equiJoinClause("ORDERS_OK", "LINEITEM_OK")),
                                 ImmutableMap.of("ORDERS_OK", "LINEITEM_OK"),
-                                Optional.empty(),
-                                tableScan("orders", ImmutableMap.of("ORDERS_OK", "orderkey")),
+                                anyTree(
```

Comment:
nit: we could use `DynamicFilters#createDynamicFilterExpression` as in  `TestRemoveUnsupportedDynamicFilters#testDynamicFilterConsumedOnBuildSide` to be little more explicit

At least please validate that there is `filter` node above table scan

Reply:
If the filter is not there then `JoinMatcher#matchDynamicFilters` should fail. The semi-join DF tests are also on the same lines. I can add the FilterNode explicitly here if that's better.

Reply:
Also, using `createDynamicFilterExpression` would require knowing the DynamicFilterId. We are able to do that there as the PlanNode is manually created in those tests.